activate.php

```diff
@@ -405,4 +408,49 @@ public function reactivate_addons() {
 			}
 		}
 	}
+
+	/**
+	 * Migrate old datas.
+	 *
+	 * @since 4.4.0
+	 */
+	public function migrate() {
+
+		// Migrate old datas according to the AP_DB_VERSION constant.
+		switch ( AP_DB_VERSION ) {
```

Comment:
@mebishalnapit for this I don't think we have to use a switch statement.

Reply:
@rahularyan It might be better to use switch statement since adding too much of if statement to check the condition for AP_DB_VERSION later on it could be bloated.

Reply:
We can later update when needed but for now if will be simple as we have only once check to make.

Reply:
Updated with `if` statement.